content/en/developers/prometheus.md

```diff
@@ -51,68 +51,91 @@ instances:
-        self.NAMESPACE = 'kubedns'
+    def __init__(self, name, init_config, instances=None):
+        METRICS_MAP = {
+            #metrics have been renamed to kubedns in kubernetes 1.6.0
+            'kubedns_kubedns_dns_response_size_bytes': 'response_size.bytes',
+            'kubedns_kubedns_dns_request_duration_seconds': 'request_duration.seconds',
+            'kubedns_kubedns_dns_request_count_total': 'request_count',
+            'kubedns_kubedns_dns_error_count_total': 'error_count',
+            'kubedns_kubedns_dns_cachemiss_count_total': 'cachemiss_count'
+        }
 ```
 
-#### Overriding `self.metrics_mapper`
+#### Define a default instance
 
-`metrics_mapper` is a dictionary where the key is the metric to capture, and the value is the corresponding metric name in Datadog.
-The reason for the override is so that metrics reported by the OpenMetrics checks are not counted as [custom metric][5]:
+A default instance is the basic configuration used for the check. The default instance should override `namespace`, `metrics`, and `prometheus_url`.
+
+
+Note that we also overrode some of the default values for certain config options in the `OpenMetricsBaseCheck`, we can expect to see better metric behavior correlation between [Prometheus and Datadog metric types][7]. 
 
 ```python
 from datadog_checks.base import OpenMetricsBaseCheck
 
 class KubeDNSCheck(OpenMetricsBaseCheck):
-    def __init__(self, name, init_config, agentConfig, instances=None):
-        super(KubeDNSCheck, self).__init__(name, init_config, agentConfig, instances)
-        self.NAMESPACE = 'kubedns'
-        self.metrics_mapper = {
+    def __init__(self, name, init_config, instances=None):
+        METRICS_MAP = {
             #metrics have been renamed to kubedns in kubernetes 1.6.0
             'kubedns_kubedns_dns_response_size_bytes': 'response_size.bytes',
             'kubedns_kubedns_dns_request_duration_seconds': 'request_duration.seconds',
             'kubedns_kubedns_dns_request_count_total': 'request_count',
             'kubedns_kubedns_dns_error_count_total': 'error_count',
             'kubedns_kubedns_dns_cachemiss_count_total': 'cachemiss_count'
         }
+        super(KubeDNSCheck, self).__init__(
+            name,
+            init_config,
+            instances,
+            default_instances={
+                'kubedns': {
+                    'prometheus_url': 'http://localhost:8404/metrics',
+                    'namespace': 'kubedns',
+                    'metrics': [METRIC_MAP],
+                    'send_histograms_buckets': True,
+                    'send_distribution_counts_as_monotonic': True,
+                    'send_distribution_sums_as_monotonic': True,
+                }
+            },
+            default_namespace='kubedns',
+        )
 ```
 
+
 #### Implementing the check method
 
+In some cases, you may want to override the `check()` function to implement additional features. 
```

Comment:
```suggestion
If you want to implement additional features, override the `check()` function. 
```
I think this is what you're trying to say, but not sure. Just trying to tighten up the vagueness of "some cases" and "may want."